src/drive/web/modules/drive/Toolbar/components/ShortcutCreationModal.jsx

```diff
@@ -50,12 +52,22 @@ const ShortcutCreationModal = ({ onClose, onCreated, displayedFolder }) => {
         url: makedURL
       }
       try {
-        await client.collection('io.cozy.files.shortcuts').create(data)
-        Alerter.success(t('Shortcut.created'))
-        if (onCreated) onCreated()
+        if (isOffline) {
+          Alerter.error('alert.offline')
+        } else {
+          await client.collection('io.cozy.files.shortcuts').create(data)
```

Comment:
We should avoid using the collection directly. Can't we use save() with the _type / doctype set to io.cozy.files.shortcuts ?

Reply:
it should be done

Reply:
I will ask another review as I am not very familiar with cozy-client.